spec/daru/io/exporters/csv_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
```

Comment:
`spec_helper` totally should not be required in each `_spec.rb` (but rather in `.rspec` config file).
And most of necessary standard libraries should be required in `spec_helper.rb`